helidon-maven-plugin/src/main/java/io/helidon/build/maven/graal/FilteringComponent.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.build.maven.graal;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import io.helidon.build.util.SourcePath;
+
+/**
+ * A configuration filtering component using excludes and includes list.
+ */
+public class FilteringComponent {
```

Comment:
How about we name this class something else, maybe just PathComponent ?

Reply:
Done